donkeycar/parts/tub_v2.py

```diff
@@ -0,0 +1,112 @@
+import json
+import os
+import time
+
+import numpy as np
+from PIL import Image
+
+from donkeycar.parts.datastore_v2 import Manifest, ManifestIterator
+
+
+class Tub(object):
+    '''
+    A datastore to store sensor data in a key, value format. \n
+    Accepts str, int, float, image_array, image, and array data types.
+    '''
+    def __init__(self, base_path, inputs=[], types=[], metadata=[], max_catalog_len=1000):
```

Comment:
Could you pleas explain more about max_catalog_len?
I get a feeling it is somekind of limitation that some people should be aware of, or am I wronh?

Reply:
No need to worry about this at all. For reference, this is the maximum number of records per catalog file.